.travis.yml

```diff
@@ -27,7 +38,7 @@ env:
     - REACT=16
 sudo: false
 matrix:
-  fast_finish: true
+  fast_finish: false
```

Comment:
why would we want to disable this? That will slow down our builds a lot.

Reply:
* The overall results (including passing) represent the overall `Test Design`, we present it based on the repo source organization. It is also used for Metric purposes, related to failures and resolution rates. Can take a look at our stuff using the `demo` link above. We generate a number of metrics based on the continuous collection of CI results. 

* I believe the `fast_finish` set to `false` is a mistake on our side